pages/workflow.md

```diff
@@ -1,54 +1,107 @@
+**Orange3**
 
----
+**Orange3** is a user friendly program that is used to implement another skill in which is required to fully develop a functioning research project. Text anaylsis is a skill that many people use and not fully understand that they are using it. Simply, text anaylsis is the deep understanding on how sources link togeather. **Orange3** simplifies this process by determining constants within different sources. 
+1. Open **Orange3** from the desktop app.
 
-To put your webscraper recipe online...
+![image]({{ site.baseurl }}/assets/images/Orange3_1.JPG)
 
-*If you post a recipe, you should also post instructions on how to import it, don't assume the reader knows!*
+2. Select the **corpus** in the text mining catagory. Upload the file. For this demonstration we are going to use the data scraped by **Webscraper.IO** off the slave voyage database.
 
-1. Open your sitemap.
-2. Click the middle menu that says "Sitemap <your sitemap name>".
-3. Click "Export Sitemap" and copy the JSON data from the form.
-4. To help readability, go to [JSON Formatter](https://jsonformatter.curiousconcept.com/).
-5. Paste the raw JSON, click "Process", then copy the output.
-6. Select and copy the output and put it into a block quote (using the triple backtick ```````).
-7. To enable syntax coloring, write the word `json` immediately afterward.
-8. To see an example, look at the raw code for the blockquote below.
-	
+![image]({{ site.baseurl }}/assets/images/Orange3_2.JPG)
+
+3. Next, place down a **corpus viewer** to make sure that the **corpus** pulled in the correct data.
+
+![image]({{ site.baseurl }}/assets/images/Orange3_3.JPG)
+
+4.Once, you have made sure that the correct data has been pulled; drop down a **bag of word**. This application will sort through your data and pick out words that show up in frequency. 
+
+![image]({{ site.baseurl }}/assets/images/Orange3_4.JPG)
+
+5. After the **bag of words** is placed drop down a **word cloud**.
+
+![image]({{ site.baseurl }}/assets/images/Orange3_5.JPG)
 
-```json
-{  
-   "_id":"fake_sitemap",
-   "startUrl":[  
-      "https://www.fake-url.com"
-   ],
-   "selectors":[  
+Once you place down **word cloud** you are able to see the like words in which appear over the document. The larger the word and the closer the word is to the center of the cloud; the more frequent the word is used throughout the document.
 
-   ]
-}
-```
+![image]({{ site.baseurl }}/assets/images/word_cloud.png)
 
+From the visualization above we can conclude that through all of the slave voyages gathered in the dataset, the majority of the voyages are Spanish in nature. Many words that show up frequently have ties to the Spanish government. This discovery does not falter away very far from popular history on the international slave trade. There was no secret that that the trans Atlantic slave trade was primarily dominated by the Spanish and the English. 
```

Comment:
Great all around approach, especially the Palladio display. I hope you add links, especially so you can see the full-size images, because that would make this workflow stellar... one that I would love to use as an example of how to do it right for future semesters.